torch/nn/modules/rnn.py

```diff
@@ -531,15 +588,24 @@ class LSTM(RNNBase):
     def __init__(self, *args, **kwargs):
         super(LSTM, self).__init__('LSTM', *args, **kwargs)
 
+    def get_expected_cell_size(self, input: Tensor, batch_sizes: Optional[Tensor]) -> Tuple[int, int, int]:
+        if batch_sizes is not None:
+            mini_batch = batch_sizes[0]
+            mini_batch = int(mini_batch)
```

Comment:
> torch/nn/modules/rnn.py:594: error: Incompatible types in assignment (expression has type "int", variable has type "Tensor")  [assignment]
torch/nn/modules/rnn.py:596: error: Incompatible types in assignment (expression has type "int", variable has type "Tensor")  [assignment]

The error message is complaining that mini_batch had type Tensor, but then we reassigned it a value of type int.
```
mini_batch = int(batch_sizes[0])
```
should fix that